osu.Game/Beatmaps/Drawables/Cards/Statistics/NominationsStatistic.cs

```diff
@@ -12,16 +14,27 @@ namespace osu.Game.Beatmaps.Drawables.Cards.Statistics
     /// </summary>
     public partial class NominationsStatistic : BeatmapCardStatistic
     {
-        private NominationsStatistic(BeatmapSetNominationStatus nominationStatus)
+        private NominationsStatistic(int current, int required)
         {
             Icon = FontAwesome.Solid.ThumbsUp;
-            Text = nominationStatus.Current.ToLocalisableString();
-            TooltipText = BeatmapsStrings.NominationsRequiredText(nominationStatus.Current.ToLocalisableString(), nominationStatus.Required.ToLocalisableString());
+            Text = current.ToLocalisableString();
+            TooltipText = BeatmapsStrings.NominationsRequiredText(current.ToLocalisableString(), required.ToLocalisableString());
         }
 
-        public static NominationsStatistic? CreateFor(IBeatmapSetOnlineInfo beatmapSetOnlineInfo)
+        public static NominationsStatistic? CreateFor(APIBeatmapSet beatmapSet)
```

Comment:
I'm not sure how much we want to be strict with using the interface instead of the direct model type. I've changed it here because I need information about the number of different rulesets in difficulties of this beatmap set.

I could approach this more appropriately by introducing a `RulesetsCount` in the interface, or not change this and perform a soft cast inside the method. Open for suggestions.